pydantic_ai_slim/pydantic_ai/_run_context.py

```diff
@@ -38,6 +38,8 @@ class RunContext(Generic[RunContextAgentDepsT]):
     """The original user prompt passed to the run."""
     messages: list[_messages.ModelMessage] = field(default_factory=list)
     """Messages exchanged in the conversation so far."""
+    validation_context: Any = None
+    """Additional Pydantic validation context for the run outputs."""
```

Comment:
Let's not call it "additional", and let's link to the Pydantic validation context docs.

Would also be good to mention this feature in docs somewhere :)